config/initializers/airbrake.rb

```diff
@@ -50,7 +50,7 @@
     # Airbrake. By default, all "password" attributes will have their contents
     # replaced.
     # https://github.com/airbrake/airbrake-ruby#blacklist_keys
```

Comment:
update that link too https://github.com/airbrake/airbrake-ruby#blocklist_keys 👍

Reply:
Done :)